client/src/js/controller/customer_controller.js

```diff
@@ -0,0 +1,41 @@
+var app = angular.module('RRS', []);
+app.controller('customerController', function($scope,$http) {
+
+    $scope.getReservation = function(){
+        var confirmationCode = $scope.confirmCode;
+        $http.get("http://localhost:8080/RRS_App/api/reservation/reservation/"+confirmationCode)
+            .then(function(response)
+                {
+                    $scope.res = response.data;
+                }
+            );
```

Comment:
only success is being handled in all the `$http` requests, also need to handle the failure cases, so update then to `then(successFn, errorFn)`